### PR TITLE
Fix Github Action pending problem

### PR DESCRIPTION
**Pull request by asdf2014**
## Purpose of this pull request

This patch is trying to fix the pending problem of Github Action.

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later

### CODE REVIEW

**Review by CalvinKirs — COMMENTED** [review]
Why do we need to display the specified jdk?

**Comment by asdf2014** [comment]
Hi @CalvinKirs . Thanks for your comment. The `jdk.version` option is not necessary, however we can use this option to ensure the version of JDK to avoid affected by environment variables.

**Comment by CalvinKirs** [comment]
> Hi @CalvinKirs . Thanks for your comment. The `jdk.version` option is not necessary, however we can use this option to ensure the version of JDK to avoid affected by environment variables.

@kezhenxu94 hi WDYT?

**Comment by leo65535** [comment]
hi @asdf2014, we should remove the hard code `jdk.version`, because we already setup the JDK version in github action.
https://github.com/apache/incubator-seatunnel/blob/dev/.github/workflows/backend.yml

It will limit us when we want to use other JDK version to build the SeaTunnel project, we can set different JDK env by using github action.

File: .github/workflows/backend.yml [hunk 1]
```diff
@@ -49,15 +49,23 @@ jobs:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven
       - name: Check Style
+        env:
+          MAVEN_OPTS: -Xmx2G -Xms2G
```
Comment (zhongjiajie) [inline .github/workflows/backend.yml:53]: Should we add this to a single line instead of each step?

**Comment by asdf2014** [comment]
Hi @leo65535 @zhongjiajie @CalvinKirs , thanks for your comments, which have been addressed, PTAL

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**